website/docs/cdktf/community.html.md

```diff
@@ -7,10 +7,16 @@ description: "Resources to help you contribute to the codebase, submit issues, a
 
 # Community
 
-CDK for Terraform is an early experimental project and the development team would love your feedback to help guide the project.
+CDK for Terraform is a rapidly-growing project and the development team would love your feedback and direct contributions to help guide the project.
+
+## Questions & Issues
+If you have questions, the HashiCorp [Discuss](https://discuss.hashicorp.com/) platform using the [terraform-cdk](https://discuss.hashicorp.com/c/terraform-core/cdk-for-terraform/) category is the best place for general discussion. If you run into issues or have ideas for improvements, please file a [bug](https://github.com/hashicorp/terraform-cdk/issues/new?assignees=&labels=bug&template=bug-report.md&title=) or request a new [feature](https://github.com/hashicorp/terraform-cdk/issues/new?assignees=&labels=enhancement&template=feature-request.md&title=). Please let us know if you'd like to talk - we are very interested in seeing how people use CDK for Terraform in practice.
+
+## Contributing
+The CDK for Terraform project is very open to community contributions. The easiest way to help is to file bugs when you run into issues, comment on existing issues if you're seeing them, and comment or upvote on feature requests if you have use cases where they would be helpful. Hearing _why_ you're interested extremely useful.
+
+If you're interested in contributing code, please look at the [CONTRIBUTING.md](./CONTRIBUTING.md) guide. Before you file a large pull request, please file an enhancement request, and tell us that you're interested in building it. We're happy to work together on design, to make sure your PR can be reviewed and accepted quickly.
```

Comment:
Is there also a way to provide a link to an enhancement request? 

```suggestion
If you're interested in contributing code, please look at the [CONTRIBUTING.md](./CONTRIBUTING.md) guide. Before you file a large pull request, please file an enhancement request to tell us what you're interested in building. We're happy to work with you on the design to make sure we can review and accept your PR quickly.
```

Reply:
@laurapacilio this is linked in the # Questions and Issues section above, so maybe just a question of where it fits best?

> request a new [feature](https://github.com/hashicorp/terraform-cdk/issues/new?assignees=&labels=enhancement&template=feature-request.md&title=)

Reply:
oh! I thought reading this that enhancement was different somehow - maybe we just say "request a new feature"? We don't need to link twice...I just think the use of enhancement confused me

Reply:
enhancement = new feature :) so the same issue template would apply. I'm flexible on the wording

Reply:
okay! I updated my comment to use feature instead - thanks!